README.md

```diff
@@ -1,8 +1,9 @@
 # infusion-icons #
-SVG icons for use in Infusion
+SVG icons which can be used directly or compiled into an icon font.
+The icons have been created based on the [Infusion Icons Visual Style Guide](https://wiki.fluidproject.org/display/fluid/Infusion+Icons+Visual+Style+Guide).
 
 ## Documentation ##
 
-The documentation containing best practices for creating the SVG icons as well as compiling them into an icon-font can be found at on the [Standard workflow in maintaining and creating icon fonts](https://wiki.fluidproject.org/display/fluid/Standard+workflow+in+maintaining+and+creating+icon+fonts) wiki page.
+The documentation containing best practices for creating the SVG icons as well as compiling them into an icon font can be found on the [Standard workflow in maintaining and creating icon fonts](https://wiki.fluidproject.org/display/fluid/Standard+workflow+in+maintaining+and+creating+icon+fonts) wiki page.
 
 <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
```

Comment:
Does the CC-BY license cover the Google content in the wiki doc? Do we need to attribute Google content differently?

Reply:
The CC-BY license is meant for this repo, that is, the SVG icons themselves. The documentation in the wiki is also covered by a CC license. Maybe this can be better distinguished in the README. In relation to the google attribution, the material guides are licensed under apache 2 ( https://material.io/guidelines/material-design/introduction.html#introduction-principles ) <--- see the bottom of that page.

Reply:
I've added a note about the apache license to https://wiki.fluidproject.org/display/fluid/Infusion+Icons+Visual+Style+Guide